python/arcticdb/version_store/library.py

```diff
@@ -940,6 +940,56 @@ def read_metadata(self, symbol: str, as_of: Optional[AsOf] = None) -> VersionedI
         """
         return self._nvs.read_metadata(symbol, as_of)
 
+    def read_metadata_batch(
+        self, symbols: List[Union[str, ReadRequest]], query_builder: Optional[QueryBuilder] = None
+    ) -> List[VersionedItem]:
+        """
+        Reads the metadata of multiple symbols.
+
+        Parameters
+        ----------
+        symbols : List[Union[str, ReadRequest]
+            List of symbols to read.
+            Params columns, date_range and query_builder from ReadRequest are not used
+        
+        query_builder: This parameter is not used
+
+        Returns
+        -------
+        List[VersionedItem]
```

Comment:
Should document what happens if a version is specified that does not exist

Reply:
done